### ISSUE TITLE
Replace pkg & Upgrade nodejs version

### ISSUE THREAD

**Issue by Hypfer** [Hypfer/Valetudo#522]
The project is currently using an old and outdated version of `zeit/pkg` as well as nodejs 10.

This approach was taken to avoid installing the full nodejs runtime on the robot and instead simply having a single binary to scp onto the robot.
Since it's severely outdated and we're also at a point where we have to modify the firmwares a lot to get Valetudo to work, it might be reasonable to drop this and simply install the node runtime in the firmware build process.

Newer nodejs versions are a huge improvement both performance- as well as memory-wise so that's definitely worth the hassle

**Comment by Hypfer** [comment]
I actually got Valetudo working using [lowjs](https://github.com/neonious/lowjs). It does require some changes though.

https://www.neonious.com/lowjs/documentation/nodejs-api.html

Since synchronous FS operations aren't implemented yet, we'd need to replace those. Thats probably a good idea anyways.

Sadly, zlib is also not implemented. [pako](https://www.npmjs.com/package/pako) could be an alternative.

Furthermore, I had to completely disable the soundfile upload feature since it requires the `multer` library which in turn requires  the `mkdirp` which does some unsupported `fs` calls.

I also had to disable the `compression` module for the webserver, since it uses undocumented internal nodejs code as seen here: https://github.com/expressjs/compression/pull/155

`dgram socket.bind()` doesn't accept `undefined` as the port value so that had to be changed as well.


Other than that it seems that it just works which is awesome.

The overall install size would be about 5mb for lowjs + 12mb for valetudo & node_modules.
